lib/log_monitor.rb

```diff
@@ -49,9 +49,24 @@ def log_access(log_entry)
 
   def check_fails(log_entry)
     logstash_tags = []
-    if known_good_fail?(log_entry)
-      statsd_sender.increment("known_good_fail.status_#{log_entry.status}")
-      logstash_tags << "known_good_fail"
+    last_success = nil
+
+    if log_entry.method == "GET"
+      # Our list of known good urls is only for GET requests, so ignore any
+      # non-GET requests for this bit of monitoring.
+      last_success = last_success_date(log_entry)
+
+      if status_code_is_failure?(log_entry)
```

Comment:
this block might be a bit clearer if it opened with:

`if status_code_is_failure?(log_entry) && last_success != nil`